pages/MoodEntry.tsx

```diff
@@ -8,15 +8,22 @@ const MoodEntry = () => {
   // State that holds selected mood
   const [mood, setMood] = useState('');
   const [comment, setComment] = useState('');
+  const [error, setError] = useState('');
```

Comment:
Here, I would make the type of `error` `string | null`. Like this:

```typescript
const [error, setError] = useState<string | null>()
```

This would let you do some conditional rendering later down, like this:

```typescript
{ error && <div className={styles.errorField}>{error}</div>
```

This means that the error message isn't a part of the DOM unless there is a message to display (a pretty common pattern for a lot of things, truth be told). You might even be able to get away with `useState<string>` and omitting the union type.